Parse/Public/ParseClient.cs

```diff
@@ -122,7 +129,7 @@ public struct Configuration {
     /// </param>
     public static void Initialize(Configuration configuration) {
       lock (mutex) {
-        HostName = HostName ?? new Uri("https://api.parse.com/1/");
```

Comment:
do we still `HostName`? (even in tests). If we don't, we need to clean that up (doesn't need to be in this PR)